src/vaadin-date-time-picker-time-picker.html

```diff
@@ -0,0 +1,54 @@
+<!--
+@license
+Copyright (c) 2019 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<link rel="import" href="../../vaadin-time-picker/src/vaadin-time-picker.html">
+<link rel="import" href="vaadin-date-time-picker-time-text-field.html">
+<link rel="import" href="vaadin-date-time-picker-helper.html">
+
+<script>
+  (function() {
+    let memoizedTemplate;
+
+    /**
+      * The time-picker element.
+      *
+      * ### Styling
+      *
+      * See [`<vaadin-time-picker>` documentation](https://github.com/vaadin/vaadin-time-picker/blob/master/src/vaadin-time-picker.html)
+      * for `<vaadin-date-time-picker-time-picker>` parts and available slots (prefix, suffix etc.)
+      *
+      * See [ThemableMixin – how to apply styles for shadow parts](https://github.com/vaadin/vaadin-themable-mixin/wiki)
+      *
+      * @memberof Vaadin
+      * @extends Vaadin.TimePickerElement
+      */
+    class DateTimePickerTimePickerElement extends Vaadin.TimePickerElement {
+      static get is() {
+        return 'vaadin-date-time-picker-time-picker';
+      }
+
+      static get template() {
+        if (!memoizedTemplate) {
+          memoizedTemplate = super.template.cloneNode(true);
+
+          memoizedTemplate.innerHTML = Vaadin.DateTimePickerHelper.replaceTagNames(
+            memoizedTemplate.innerHTML,
+            'vaadin-time-picker-text-field',
+            'vaadin-date-time-picker-time-text-field'
+          );
+        }
+        return memoizedTemplate;
+      }
+
+      // Overridden only for fixing the tag name used in the querySelector
+      get __inputElement() {
+        return this.__memoInput || (this.__memoInput = this.shadowRoot.querySelector('vaadin-date-time-picker-time-text-field'));
+      }
```

Comment:
The only test failure is caused by polymer lint giving this warning from here:
"src/vaadin-date-time-picker-time-picker.html(47,7) warning [overriding-private] - Overriding private member '__inputElement' inherited from Vaadin.TimePickerElement"

I'm not sure how to get around this. I found no way to disable polymer lint for a specific line (something like `eslint-disable-line` for eslint doesn't seem to exist for polymer lint).  Do we really need to make some change in `vaadin-time-picker` repo for fixing this? Without this method there's a JS error on page load because it can't find the `<vaadin-date-time-picker-time-text-field>` element since here it has been renamed/extended.

Reply:
Implemented a change in time picker to make this easier https://github.com/vaadin/vaadin-time-picker/pull/119. Just need to release a new patch version of time picker now and then refactor this part of this PR.

Reply:
This part was refactored to solve the issue by overriding the new protected method `_getInputElement()` introduced in `vaadin-time-picker` [v2.0.3](https://github.com/vaadin/vaadin-time-picker/releases/tag/v2.0.3)